Data/symbol-properties/symbol-properties-database.csv

```diff
@@ -170,9 +175,12 @@ cme,MJY,future,Micro Japanese Yen/U.S. Dollar (JPY/USD) Futures,USD,1250000,1e-0
 cme,MNH,future,Micro USD/CNH Futures,USD,10000,0.0001,1
 cme,MNQ,future,Micro E-mini Nasdaq-100 Index Futures,USD,2,0.25,1,,1
 cme,MSF,future,Micro Swiss Franc/U.S. Dollar (CHF/USD) Futures,USD,12500,0.0001,1,,1
+cme,NIY,future,Nikkei/YEN Futures,JPY,500.0,500.0,1.0
 cme,NKD,future,Nikkei/USD Futures,USD,5.0,5.0,1.0
 cme,NQ,future,E-mini Nasdaq-100 Futures,USD,20.0,0.25,1.0
 cme,RTY,future,E-mini  Russell 2000 Index Futures,USD,50.0,0.1,1.0
+cme,TPD,future,TOPIX USD Futures,USD,50.0,50.0,1.0
+cme,TPY,future,TOPIX JPY Futures,JPY,5000.0,5000.0,1.0
```

Comment:
Don't think the minimum price variation column is right? See `MINIMUM PRICE FLUCTUATION` in CME specs, can compare with other existing futures 👍